certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+									}
+								},
+								"value-path": "Schema"
+							}
+						}
+					}
+				}
+			},
+			"put": {
+				"summary": "Update an item",
+				"operationId": "UpdateListItem",
+				"description": "Update an item",
+				"x-ms-summary": "Update an item",
+				"tags": [
+					"lists"
+				],
+				"security": [
+					{
+						"oauth2_auth": []
+					}
+				],
+				"parameters": [
+					{
+						"name": "listName-dynamic",
+						"type": "string",
+						"in": "path",
+						"description": "Select the list you want to update",
+						"required": true,
+						"x-ms-summary": "Select List",
+						"x-ms-url-encoding":"single",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					},
+					{
+						"in": "path",
+						"name": "itemId",
+						"required": true,
+						"description": "Numeric ID of the item to update",
+						"type": "integer",
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Item Id"
+					},
+					{
+						"x-ms-summary": "Item Profile",
+						"name": "dynamicListSchema",
+						"in": "body",
+						"description": "Dynamic schema of selected list",
+						"schema": {
+							"type": "object",
+							"x-ms-dynamic-schema": {
+								"operationId": "GetSingleItemSchema",
+								"parameters": {
+									"listName": {
+										"parameter": "listName-dynamic"
+									}
+								},
+								"value-path": "Schema"
+							}
+						}
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "The item was updated successfully"
+					}
+				}
+			}
+		},
+		"/lists/{listName-dynamic}/items/{itemId}/inherited-links": {
+			"get": {
+				"summary": "Get a inherit links for a given item",
+				"x-ms-summary": "Get a inherit links for a given item",
+				"description": "Gets a list of items that expect to be linked to the current item due to the inherit link configuration on your RAPID site.",
+				"operationId": "GetInheritLinks",
+				"security": [
+					{
+						"oauth2_auth": []
+					}
+				],
+				"parameters": [
+					{
+						"name": "listName-dynamic",
+						"type": "string",
+						"in": "path",
+						"description": "Select the Table you want to get items from.",
+						"required": true,
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Select Source Table",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					},
+					{
+						"in": "path",
+						"name": "itemId",
+						"required": true,
+						"description": "Numeric ID of the item to get",
+						"type": "integer",
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Item Id"
+					},
+					{
+						"name": "type",
+						"type": "string",
+						"in": "query",
+						"description": "Select the Table you want to get items for.",
+						"required": true,
+						"x-ms-summary": "Select Target Table",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "XXX SOMETHING",
```

Comment:
@MattFranklin-Rapid Can you please update it with valid response description.

Reply:
Added missing template response description to inherit-links action